backend/src/akvo/lumen/component/keycloak.clj

```diff
@@ -220,11 +258,79 @@
           (println (format "Tried to remove user: %s" user-id))
           (lib/internal-server-error))))))
 
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; API Authorization
+;;;
+
+(defn- api-get-async [{:keys [timeout] :as req-opts} url]
```

Comment:
We already have this logic in fetch-openid-configuration and request-headers. Maybe we can move it to some helper fn?